sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/_conversation_analysis_client.py

```diff
@@ -0,0 +1,98 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from copy import deepcopy
+from typing import TYPE_CHECKING
+
+from azure.core import PipelineClient
+from msrest import Deserializer, Serializer
+
+from . import models
+from ._configuration import ConversationAnalysisClientConfiguration
+from .operations import ConversationAnalysisClientOperationsMixin
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from typing import Any, Optional
+
+    from azure.core.credentials import AzureKeyCredential
+    from azure.core.rest import HttpRequest, HttpResponse
+
+class ConversationAnalysisClient(ConversationAnalysisClientOperationsMixin):
+    """This API accepts a request and mediates among multiple language projects, such as LUIS Generally Available, Question Answering, LUIS Deepstack, and then calls the best candidate service to handle the request. At last, it returns a response with the candidate service's response as a payload.
+
+ In some cases, this API needs to forward requests and responses between the caller and an upstream service.
+
+    :param endpoint: Supported Cognitive Services endpoint (e.g.,
+     https://:code:`<resource-name>`.api.cognitiveservices.azure.com).
+    :type endpoint: str
+    :param credential: Credential needed for the client to connect to Azure.
+    :type credential: ~azure.core.credentials.AzureKeyCredential
+    """
+
+    def __init__(
+        self,
+        endpoint,  # type: str
+        credential,  # type: AzureKeyCredential
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+        _endpoint = '{Endpoint}/language'
+        self._config = ConversationAnalysisClientConfiguration(endpoint, credential, **kwargs)
+        self._client = PipelineClient(base_url=_endpoint, config=self._config, **kwargs)
+
+        client_models = {k: v for k, v in models.__dict__.items() if isinstance(v, type)}
+        self._serialize = Serializer(client_models)
+        self._deserialize = Deserializer(client_models)
+        self._serialize.client_side_validation = False
+
+
+    def send_request(
+        self,
+        request,  # type: HttpRequest
```

Comment:
This seems rather LLC-ish. I thought we were doing value-add methods here, or is this Python's fallback support for LLCs on every client (something .NET sorely lacks).

// @iscai-msft @johanste

Reply:
@heaths this is by design. Going forward, we want all of our python SDKs, regardless of how much convenience they have, to have a `send_request` method to break the glass

Reply:
I did add an AnalyzeConversation (singular, note, after internal discussion - their operationId is plural even though you can only analyze one utterance) method as seen here: https://github.com/Azure/azure-sdk-for-net/pull/24030

I also had to rename a couple models but requested that the swagger do so.